Source/Core/Core/AchievementManager.cpp

```diff
@@ -386,8 +387,8 @@ bool AchievementManager::IsHardcoreModeActive() const
 }
 
 template <typename T>
-void AchievementManager::FilterApprovedIni(std::vector<T>& codes,
-                                           const std::string& game_ini_id) const
+void AchievementManager::FilterApprovedIni(std::vector<T>& codes, const std::string& game_ini_id,
+                                           const u16& revision) const
```

Comment:
```suggestion
                                           u16 revision) const
```

A simple integer is faster to pass as a copy than as a reference, so there's no reason to use a reference here.

(The same applies to other functions edited in this PR.)